airbyte-integrations/connectors/source-recurly/source_recurly/schemas/accounts.json

```diff
@@ -140,6 +153,30 @@
     "deleted_at": {
       "type": ["null", "string"],
       "format": "date-time"
+    },
+    "billing_info": {
+      "$ref": "billing_infos.json"
+    },
+    "external_accounts": {
```

Comment:
Hmmm, how do I best understand schema changes? These look like new fields to me, but you're saying we haven't changed the schemas, no?

Reply:
Yes, sorry my note may have been a bit unclear! There are indeed a whole bunch of added fields to the schemas, but we only consider the changes to be breaking if an existing field is removed, renamed or given a new type. 

The "$ref" key allows us to reference the schemas in the connector's /schemas/shared folder, so it's very helpful when nested objects like "billing_infos" get reused in multiple streams, as is the case for a lot of Recurly's streams